Formula/xdelta.rb

```diff
@@ -12,9 +12,25 @@ class Xdelta < Formula
     sha256 "7adf5ae7a00473f5c12f8c377da22ad3f98a0ef4e179c6c0b64b03de075cc756" => :mountain_lion
   end
 
+  depends_on "xz"
+  depends_on "libtool" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
   def install
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
-    system "make", "install"
+    args = %W[
```

Comment:
Oops, my mistake. I was originally playing around with manipulating the arguments list but undid those changes.